docs/products/postgresql/howto/manage-pool.rst

```diff
@@ -0,0 +1,52 @@
+Manage Connection Pooling
+=========================
+
+Connection Pooling allows you to maintain very large numbers of connections to a database while minimizing the consumption of server resources. Read more about it on :doc:`../concepts/pg-connection-pooling`.
+
+1. Log in to the Aiven web console and select your PostgreSQL service.
+
+   You can connect directly to the PostgreSQL server using the settings listed on the *Overview* page (the *Service URI* address), but this type of connection does not use PGBouncer pooling.
+
+
+2. Click the **Pools** tab.
+
+   This opens a list of the PGBouncer connection pools defined for the service.
+
+
+3. Click **Add Pool**.
```

Comment:
This post is titled "Management" and now we're assuming the user wants to add. Let's just say "here you can can see the list, add or remove pools. The settings available are:

Reply:
Fixed